src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultHybridCache.StampedeStateT.cs

```diff
@@ -175,29 +176,64 @@ private async Task BackgroundFetchAsync()
                 // nothing from L2; invoke the underlying data store
                 if ((Key.Flags & HybridCacheEntryFlags.DisableUnderlyingData) == 0)
                 {
-                    var cacheItem = SetResult(await _underlying!(_state!, SharedToken).ConfigureAwait(false));
-
-                    // note that at this point we've already released most or all of the waiting callers; everything
-                    // else here is background
-
-                    // write to L2 if appropriate
-                    if ((Key.Flags & HybridCacheEntryFlags.DisableDistributedCacheWrite) == 0)
+                    // invoke the callback supplied by the caller
+                    var newValue = await _underlying!(_state!, SharedToken).ConfigureAwait(false);
```

Comment:
As per our coding style guide, we don't use `var` unless the type is obvious.
https://github.com/dotnet/extensions/blob/main/CONTRIBUTING.md#dos-and-donts

Reply:
changed in 30f96c60e8ce49f07b37e8ae89505f25865ef15d